vendor/purs/runtime.ss

```diff
@@ -108,4 +109,15 @@
     (lambda (x) (lambda (xs) (cons x xs))))
 
 
+  ;
+  ; Runtime failures
+  ;
+
+  (define (fail)
+    (raise
```

Comment:
I sometimes see crashes about `&non-continuable`, and while I am not sure where they are from, I suspect it's because we use `raise` instead of `raise-continuable` - from the [docs](https://scheme.com/tspl4/exceptions.html#./exceptions:s3):

>  If an exception handler attempts to return to the continuation of a non-continuable exception, however, a new exception with condition type `&non-continuable` is raised

Should this one be a `raise-continuable`?